azkaban-common/src/main/java/azkaban/imagemgmt/daos/ImageRampupDaoImpl.java

```diff
@@ -0,0 +1,491 @@
+/*
+ * Copyright 2020 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package azkaban.imagemgmt.daos;
+
+import azkaban.db.DatabaseOperator;
+import azkaban.db.SQLTransaction;
+import azkaban.imagemgmt.exeception.ImageMgmtDaoException;
+import azkaban.imagemgmt.exeception.ImageMgmtException;
+import azkaban.imagemgmt.models.ImageRampup;
+import azkaban.imagemgmt.models.ImageRampup.StabilityTag;
+import azkaban.imagemgmt.models.ImageRampupPlan;
+import azkaban.imagemgmt.models.ImageRampupPlanRequest;
+import azkaban.imagemgmt.models.ImageRampupRequest;
+import azkaban.imagemgmt.models.ImageType;
+import azkaban.imagemgmt.models.ImageVersion;
+import azkaban.imagemgmt.models.ImageVersion.State;
+import com.google.common.collect.Iterables;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import org.apache.commons.dbutils.ResultSetHandler;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * DAO Implementation for accessing image rampup plan and rampups for an image type. This class
+ * contains implementation for methods such as create, get, update etc. for image rampups.
+ */
+@Singleton
+public class ImageRampupDaoImpl implements ImageRampupDao {
+
+  private static final Logger log = LoggerFactory.getLogger(ImageRampupDaoImpl.class);
+
+  private final DatabaseOperator databaseOperator;
+
+  private final ImageTypeDao imageTypeDao;
+
+  private final ImageVersionDao imageVersionDao;
+
+  private static String INSERT_IMAGE_RAMPUP_PLAN_QUERY = "insert into image_rampup_plan ( name, "
+      + "description, type_id, active, created_by, created_on, modified_by, modified_on) "
```

Comment:
All the tables names are having prefix image, hence none of the columns are having image prefix as it will be redundant. This convention is followed for all the tables and associated columns.